### DISCUSSION TITLE
Does signoz support create account ?

### DISCUSSION THREAD

**Discussion by monrah1990** [SigNoz/signoz#2091] [Q&A] [answered]
Hi

Does signoz support add users?

**Comment by monrah1990** [comment]
![signoz](https://user-images.githubusercontent.com/82679080/213919801-acfba619-a1fc-4a2e-8c56-0c51e26ace45.png)

**Comment by monrah1990** [comment]
as you see in the screenshot, I cant press "Get Started" and I didn't see any error.

**Comment by pranay01** [comment]
@monirerahmani-devops This should work. Can you try a simple password (like not involving any alphanumberica characters etc)

Also, which OS have you installed SigNoz in?

**Reply by monrah1990** [reply]
Thank you for your response.

Yes, I have also tried a simple password.
On Debian 11, I installed signoz.

**Comment by pranay01** [comment]
Also, tracking an issue here for this -  https://github.com/SigNoz/signoz/issues/2092

**Comment by monrah1990** [comment]
Here are the steps I took to fix my issue:

Remove "disable" from Inspect

It would be better if you fixed this

**Comment by monrah1990** [comment]
![sig1](https://user-images.githubusercontent.com/82679080/214005326-a614a646-de96-44f2-b27d-79049cfebe73.png)
![sig2](https://user-images.githubusercontent.com/82679080/214005344-6a8ad93e-0f84-4cfb-9fdf-f595ce35ba1a.png)

**Comment by monrah1990** [comment, marked as answer]
This issue was fixed in version 0.15.0.
Thank you

**Reply by pranay01** [reply]
great to hear that @monirerahmani-devops
